src/data/baseline.py

```diff
@@ -64,13 +63,25 @@ def __getitem__(self, index):
                 content = ''.join([i for i in content if i in allowed_symbols])
             pil_content = draw_one(content)
             img_content = np.array(pil_content)
-            img_content = cv2.resize(img_content, img_size) 
+            img_content = cv2.resize(img_content, img_size)
 
             img_content = img_content * 1.0 / 255
             img_content = torch.from_numpy(np.transpose(img_content[:, :, [2, 1, 0]], (2, 0, 1))).float()
 
-            return img_style, img_content, content
+            content_style = self.words[self.style_files[index].stem]
+            content_style = ''.join([i for i in content_style if i in allowed_symbols])
+            if not content_style:
+                content_style = 'o'
+            pil_content_style = draw_one(content_style)
+            img_content_style = np.array(pil_content_style)
+            img_content_style = cv2.resize(img_content_style, img_size)
+
+            img_content_style = img_content_style * 1.0 / 255
+            img_content_style = torch.from_numpy(np.transpose(img_content_style[:, :, [2, 1, 0]], (2, 0, 1))).float()
+
+            return img_style, img_content, content, img_content_style
 
         except Exception as e:
+            raise e
```

Comment:
Let's have it after logger message maybe? Also in this way lines after are not needed.